asserts/database.go

```diff
@@ -30,22 +30,19 @@ import (
 )
 
 // Backstore is a backstore for assertions. It can store and retrieve
-// assertions by type under primary paths (tuples of strings). Plus it
-// supports more general searches.
+// assertions by type under unique primary key headers. Plus it supports searching by headers.
 type Backstore interface {
-	// Put stores an assertion under the given unique primaryPath.
+	// Put stores an assertion for the given primaryKeyHeaders forming a unique key.
 	// It is responsible for checking that assert is newer than a
 	// previously stored revision.
-	Put(assertType AssertionType, primaryPath []string, assert Assertion) error
-	// Get loads an assertion with the given unique primaryPath.
+	Put(assertType AssertionType, primaryKeyHeaders []string, assert Assertion) error
+	// Get loads the assertion with the given unique key for the primaryKeyHeaders.
```

Comment:
again I don't see the need for the extra verb; "Get the assertion" works fine (and the fact that it works fine means it's the right name for the method).